graphdatascience/tests/integration/test_graph_construct.py

```diff
@@ -146,6 +146,54 @@ def test_imdb_graph_with_arrow(gds: GraphDataScience) -> None:
         G.drop()
 
 
+@pytest.mark.filterwarnings("ignore: GDS Enterprise users can use Apache Arrow")
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 3, 0))
+def test_lastfm_graph_without_arrow(gds_without_arrow: GraphDataScience) -> None:
+    G = gds_without_arrow.graph.load_lastfm()
+
+    try:
+        assert G.node_count() == 19914
+        assert G.relationship_count() == 584060
```

Comment:
I'd like to assert on more than just the counts. Let's add the stuff we put in the docs:

| name                    | lastfm
| node_count              | 19_914
| relationship_count      | 584_060
| node_labels             | ['Artist', 'User']
| relationship_types      | ['TAGGED', 'LISTEN_TO', 'IS_FRIEND']
| node_properties         | Artist: [rawId], User: [rawId]
| relationship_properties | TAGGED: [year, month, day, tagID, timestamp], LISTEN_TO: [weight], IS_FRIEND: []